[{"user_id": 16, "stars": [], "topic_id": 37103, "date_created": 1306528359.587431, "message": "I'm working on pushing out some new backend code that could affect the Event JSON feeds.  Our intention is not to break anything, so PLEASE report any problems as soon as possible.", "group_id": 591, "id": 1208639}, {"user_id": 16, "stars": [{"date_created": 1307142316.589057, "user_id": 2320}], "topic_id": 37103, "date_created": 1307141060.5629079, "message": "http://rooktopia.files.wordpress.com/2011/03/flawless-victory.jpg", "group_id": 591, "id": 1282294}, {"user_id": 1767, "stars": [], "topic_id": 37103, "date_created": 1307185613.2026429, "message": "Don't want to be a buzzkill here, but this (https://github.com/dbloete/ioctocat/issues/102) maybe related to the update.", "group_id": 591, "id": 1285593}, {"user_id": 33914, "stars": [], "topic_id": 37103, "date_created": 1307188026.8679869, "message": "As I said on github, it seems that update time is in UTC despite of \"-07:00\" postfix. Although only atom feed is affected by this issue.", "group_id": 591, "id": 1285738}, {"user_id": 32520, "stars": [], "topic_id": 37103, "date_created": 1307195407.6139369, "message": "hahahaha I had really laughed a lot with the image =]", "group_id": 591, "id": 1286393}, {"user_id": 2054, "stars": [], "topic_id": 37103, "date_created": 1307199058.7883921, "message": "This explains the JSON parsing troubles users of my app started encountering all of a sudden. I guess it's obvious I should follow this Convore group more closely.", "group_id": 591, "id": 1286536}, {"user_id": 8391, "stars": [], "topic_id": 37103, "date_created": 1307256309.6059079, "message": "There was a slight date format change that I only get when querying an org as a user.", "group_id": 591, "id": 1291955}, {"user_id": 8391, "stars": [], "topic_id": 37103, "date_created": 1307256343.9872861, "message": "I haven't checked to see what the actual difference is, but my parser doesn't like it anymore, and does for regular users.", "group_id": 591, "id": 1291956}, {"user_id": 33289, "stars": [], "topic_id": 37103, "date_created": 1307256357.1527021, "message": "so when does this get fixed? every morning my feed shows \"right now right now right now\"", "group_id": 591, "id": 1291959}, {"user_id": 35195, "stars": [], "topic_id": 37103, "date_created": 1307358547.920357, "message": "Any update on this?", "group_id": 591, "id": 1302003}, {"user_id": 33914, "stars": [], "topic_id": 37103, "date_created": 1307361143.156095, "message": "Be patient. Right now it's monday, 4:50AM at the GitHub HQ. I'm sure they're going to address this issue today. :)", "group_id": 591, "id": 1302264}, {"user_id": 33914, "stars": [], "topic_id": 37103, "date_created": 1307455176.0931189, "message": "@technoweenie we need your help down there!", "group_id": 591, "id": 1321649}, {"user_id": 16, "stars": [{"date_created": 1307501857.7302811, "user_id": 8391}], "topic_id": 37103, "date_created": 1307474637.6029861, "message": "Thanks for the heads up.  Sorry for the breakage.  Time formatting is a definite sore spot on the current API.  The new event feed api is consistent in this regard, but it's quite not ready.", "group_id": 591, "id": 1326566}, {"user_id": 33914, "stars": [], "topic_id": 37103, "date_created": 1307479164.810087, "message": "Thanks for the fix, @technoweenie! (but apparently time format change crashes iOctocat, need to push update ASAP :P) While we're at fixing stuff: private atom feed for organizations doesn't work if you use basic authentication (login/pass) instead of token in query string. I don't think it's expected behavior since user's private feed works with both methods.", "group_id": 591, "id": 1327620}, {"user_id": 1767, "stars": [], "topic_id": 37103, "date_created": 1307484727.2813089, "message": "Feeds could use versioning too. That would prevent breaking apps. :)", "group_id": 591, "id": 1328903}, {"user_id": 33289, "stars": [], "topic_id": 37103, "date_created": 1307497584.037962, "message": "yep. ioctocat doesn't even open now -- crashing every time", "group_id": 591, "id": 1331955}, {"user_id": 16, "stars": [], "topic_id": 37103, "date_created": 1307523796.202409, "message": "CvX: The time format shouldn't have changed.  I'll need specifics.  I didn't do anything with authentication in the feeds, so it's likely private org feeds never supported login/pass", "group_id": 591, "id": 1335084}, {"user_id": 16, "stars": [{"date_created": 1307533469.106262, "user_id": 651}], "topic_id": 37103, "date_created": 1307523864.0203471, "message": "@koraktor versioning doesn't help if I inadvertently change something.  There'll be new JSON/Atom APIs to go with API v3 though.", "group_id": 591, "id": 1335093}, {"user_id": 1767, "stars": [], "topic_id": 37103, "date_created": 1307525207.7926049, "message": "No problem, I think you just made iOctocat a bit more robust. ;)", "group_id": 591, "id": 1335191}, {"user_id": 1767, "stars": [], "topic_id": 37103, "date_created": 1307524153.9698861, "message": "@technoweenie Couldn't it be rolled back then?", "group_id": 591, "id": 1335117}, {"user_id": 33914, "stars": [], "topic_id": 37103, "date_created": 1307526445.6983891, "message": "@technoweenie previous date format was like this (example taken from api v2 docs): \"2010-12-09T13:50:17-08:00\" while new one uses \"Z\" designator for UTC zone. Private org feeds issue is unrelated to recent changes, but I guess it's an api inconsistency and also forces us to use user-unfriendly api tokens. :)", "group_id": 591, "id": 1335246}, {"user_id": 16, "stars": [], "topic_id": 37103, "date_created": 1307524773.8565969, "message": "Source control gives me that ability.  Though, in this case I completely replaced the backend code for events, and removed the old code after 3 days w/o any errors reported.  Doh :)", "group_id": 591, "id": 1335165}, {"user_id": 35195, "stars": [], "topic_id": 37103, "date_created": 1307537853.6061261, "message": "Thanks. This is fixed in iOctocat now - let's hope Apple accepts the new version soon.", "group_id": 591, "id": 1336691}, {"user_id": 33289, "stars": [], "topic_id": 37103, "date_created": 1308797419.7544751, "message": "2 weeks later. is apple still sitting on this?", "group_id": 591, "id": 1462618}, {"user_id": 35195, "stars": [], "topic_id": 37103, "date_created": 1308814998.9638579, "message": "The app got rejected once because of a login issue (not related to the crashes but to some changes I made, my fault) - it should be ready soon as I requested an expedited review now.", "group_id": 591, "id": 1464364}]